### DISCUSSION TITLE
Entire row in list mode as link instead of just icon and text

### DISCUSSION THREAD

**Discussion by MikeLBN** [rejetto/hfs#721] [Q&A] [answered]
Hello.

I'd like to have entire row in list mode being a link instead of just icon and text as link.
How can that be achieved?

Thanks in Advance

**Comment by rejetto** [comment]
you are wrong, the icon is part of the link, with the text. I just verified.

**Reply by PH5HP** [reply]
I think he ment that he wants the whole row as a link ... The row of the table...

**Comment by MikeLBN** [comment]
Thanks for quick response, but it's not what i meant. As of now icon and text are clickable link (marked with violet on the screenshot).
I want entire row to be a clickable link (marked with darker hue on screenshot)
![Screen Shot 2024-08-30 at 23 33 10](https://github.com/user-attachments/assets/286a5eac-6ca5-4e02-b9b4-dd8dc66f8987)

**Comment by rejetto** [comment, marked as answer]
listen the click on the row, and then simulate the click on the link, with this code in Admin-panel > Custom HTML > Html Head
```html
<script>
document.addEventListener('click', ({ target: t }) => {
  if (t.tagName === 'LI' && t.closest('ul.dir'))
      t.querySelector('a')?.click()
})
</script>
```

**Reply by MikeLBN** [reply]
Perfect! Thank You!

**Reply by PH5HP** [reply]
Hi rejetto,

Maybe an idea to gather all the code hacks somewhere to make it easier to find those?

Kind regards,
Henk - https://ph5hp.nl - https://fileserver.ph5hp.nl

**Comment by rejetto** [comment]
i'm already collecting some at https://github.com/rejetto/hfs/wiki/Customization#examples

at the moment it's just a section in the page, but as it grows it can become a dedicated page, or even a collection of pages.
wiki can be contributed by users.
